client/settings/fraud-protection/advanced-settings/rule-card-notice.js

```diff
@@ -1,121 +1,73 @@
-		case 'error':
-			return (
-				<svg
-					width="18"
-					height="18"
-					viewBox="0 0 18 18"
-					fill="none"
-					xmlns="http://www.w3.org/2000/svg"
-					data-testid="rule-card-notice-error-icon-svg"
-				>
-					<path
-						fillRule="evenodd"
-						clipRule="evenodd"
-						d={
-							'M9 1.75C4.99594 1.75 1.75 4.99594 1.75 9C1.75 13.0041 4.99594 16.25 9 16.25C13.0041 ' +
-							'16.25 16.25 13.0041 16.25 9C16.25 4.99594 13.0041 1.75 9 1.75ZM0.25 9C0.25 4.16751 ' +
-							'4.16751 0.25 9 0.25C13.8325 0.25 17.75 4.16751 17.75 9C17.75 13.8325 13.8325 17.75 ' +
-							'9 17.75C4.16751 17.75 0.25 13.8325 0.25 9Z'
-						}
-						fill="#CC1818"
-					/>
-					<path d="M10 4H8V10H10V4Z" fill="#CC1818" />
-					<path d="M10 12H8V14H10V12Z" fill="#CC1818" />
-				</svg>
-			);
-		case 'warning':
-			return (
-				<svg
-					width="18"
-					height="18"
-					viewBox="0 0 18 18"
-					fill="none"
-					xmlns="http://www.w3.org/2000/svg"
-					data-testid="rule-card-notice-warning-icon-svg"
-				>
-					<path
-						fillRule="evenodd"
-						clipRule="evenodd"
-						d={
-							'M9 1.75C4.99594 1.75 1.75 4.99594 1.75 9C1.75 13.0041 4.99594 16.25 9 16.25C13.0041 ' +
-							'16.25 16.25 13.0041 16.25 9C16.25 4.99594 13.0041 1.75 9 1.75ZM0.25 9C0.25 4.16751 ' +
-							'4.16751 0.25 9 0.25C13.8325 0.25 17.75 4.16751 17.75 9C17.75 13.8325 13.8325 17.75 ' +
-							'9 17.75C4.16751 17.75 0.25 13.8325 0.25 9Z'
-						}
-						fill="#bd8600"
-					/>
-					<path d="M10 4H8V10H10V4Z" fill="#bd8600" />
-					<path d="M10 12H8V14H10V12Z" fill="#bd8600" />
-				</svg>
-			);
-		case 'info':
-			return (
-				<svg
-					width="24"
-					height="24"
-					viewBox="0 0 24 24"
-					fill="none"
-					xmlns="http://www.w3.org/2000/svg"
-					data-testid="rule-card-notice-info-icon-svg"
-				>
-					<circle
-						cx="12"
-						cy="9"
-						r="5"
-						stroke="#007CBA"
-						strokeWidth="1.42857"
-					/>
-					<line
-						x1="9"
-						y1="16.8818"
-						x2="15"
-						y2="16.8818"
-						stroke="#007CBA"
-						strokeWidth="1.5"
-					/>
-					<line
-						x1="10"
-						y1="19.25"
-						x2="14"
-						y2="19.25"
-						stroke="#007CBA"
-						strokeWidth="1.5"
-					/>
-				</svg>
-			);
-	}
+const getInfoSVGIcon = () => {
+	return (
+		<svg
+			width="24"
+			height="24"
+			viewBox="0 0 24 24"
+			fill="none"
+			xmlns="http://www.w3.org/2000/svg"
+			data-testid="rule-card-notice-info-icon-svg"
+		>
+			<circle
+				cx="12"
+				cy="9"
+				r="5"
+				stroke="#007CBA"
+				strokeWidth="1.42857"
+			/>
+			<line
+				x1="9"
+				y1="16.8818"
+				x2="15"
+				y2="16.8818"
+				stroke="#007CBA"
+				strokeWidth="1.5"
+			/>
+			<line
+				x1="10"
+				y1="19.25"
+				x2="14"
+				y2="19.25"
+				stroke="#007CBA"
+				strokeWidth="1.5"
+			/>
+		</svg>
+	);
 };
 
 const FraudProtectionRuleCardNotice = ( { type, children } ) => {
+	let icon = <NoticeOutlineIcon />;
+
+	if ( 'info' === type ) {
+		icon = getInfoSVGIcon();
+	}
```

Comment:
Yes, looks like the TipIcon resides in 

```
import TipIcon from 'wcpay/icons';
```

so we can import it and use it.

```suggestion
	if ( 'info' === type ) {
		icon = <TipIcon />;
	}
```
Looks like this:
<img width="965" alt="image" src="https://user-images.githubusercontent.com/3295/226316911-c34620aa-729b-4ce4-98ed-76588dce4271.png">
(May need some CSS tweaks)

Reply:
Oh neat. I'll update it to use that instead.